index.html

```diff
@@ -31,6 +31,5 @@ <h1 class="title">Today's To Do</h1>
        
     </main>
    
-   
 </body>
 </html>
```

Comment:
Great job @David-Kasilia 🤝

- [ ] You may kindly remove this `index.html` file as it is not being served to development.

Reply:
I have implemented that. Thank you